bot/exts/utils/thread_bumper.py

```diff
@@ -24,6 +24,23 @@ class ThreadBumper(commands.Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    async def thread_exists_in_site(self, thread_id: int) -> bool:
+        """Return whether the given thread_id exists in the site api's bump list."""
+        # If the thread exists, site returns a 204 with no content.
```

Comment:
```suggestion
        # If the thread exists, site returns 204 No Content.
```
I was a little confused and had to search up what the 204 status code meant. This may or may not be an improvement, feel free to ignore.